x-pack/plugins/rule_registry/server/utils/create_persistence_rule_type_wrapper.test.ts

```diff
@@ -0,0 +1,292 @@
+/*
```

Comment:
Took and adapted these tests from x-pack/plugins/rule_registry/server/utils/create_lifecycle_rule_type.test.ts